dev/plugins/hu.elte.txtuml.xtxtuml/src/hu/elte/txtuml/xtxtuml/validation/XtxtUMLExternalityHelper.xtend

```diff
@@ -0,0 +1,63 @@
+package hu.elte.txtuml.xtxtuml.validation;
+
+import hu.elte.txtuml.xtxtuml.xtxtUML.TUAttribute
+import hu.elte.txtuml.xtxtuml.xtxtUML.TUConstructor
+import hu.elte.txtuml.xtxtuml.xtxtUML.TUExternality
+import hu.elte.txtuml.xtxtuml.xtxtUML.TUModifiers
+import hu.elte.txtuml.xtxtuml.xtxtUML.TUOperation
+import java.util.HashMap
+import java.util.Map
+import org.eclipse.emf.ecore.EObject
+import org.eclipse.xtext.xbase.XBlockExpression
+
+import static hu.elte.txtuml.xtxtuml.xtxtUML.TUExternality.*
+
+class XtxtUMLExternalityHelper {
+
+	val Map<EObject, TUExternality> externality = new HashMap
```

Comment:
Fixed in https://github.com/ELTE-Soft/txtUML/pull/525/commits/380647e8eb26d65b0722583fe3c983c2a9797a7b.